### PR TITLE
Updates for installing selenium libraries

### PR DESCRIPTION
**Pull request by titusfortner**
This PR is in draft because it is just english for now, will update others after this approved.

This also brings up the consideration of how do we flag the non-English versions for being out of date with English versions? Updating the code is easy, but do we just leave the text alone?

This PR:
1. For simplicity I think we should always recommend users require the full `selenium-java`, I've seen too many poms that require different things in different places that have gotten out of sync on versions, and I don't see any advantage to this for novice users.
2. Added Gradle example

### CODE REVIEW

File: website_and_docs/content/documentation/getting_started/installing_selenium_libraries.en.md [hunk 1]
```diff
@@ -11,41 +11,23 @@ First you need to install the Selenium bindings for your automation project.
...
 
-```xml
-<dependency>
-  <groupId>org.seleniumhq.selenium</groupId>
-  <artifactId>selenium-chrome-driver</artifactId>
-  <version>4.X</version>
-</dependency>
+```text
+dependencies {
+    compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.4.0'
```
Comment (diemol) [inline website_and_docs/content/documentation/getting_started/installing_selenium_libraries.en.md]:
```suggestion
    compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '4.0.0'
```

**Review by diemol — DISMISSED** [review]
Maybe add a sentence on top of the page saying "Check the downloads page to see what is the latest version?", since we are using `4.0.0` in the examples.

Also, could you please replace the content of the translated pages and add a notification on top asking for translation contributions?

**Comment by titusfortner** [comment]
Made the change, but rebased off of #861, so moving this to draft until that's merged.

**Review by diemol — APPROVED** [review]

File: website_and_docs/content/documentation/webdriver/getting_started/install_selenium_library.pt-br.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 title: "Instalando bibliotecas do Selenium"
 linkTitle: "Instalando bibliotecas do Selenium"
 weight: 1
+needsTranslation: true
 description: >
   Setting up the Selenium library for your favourite programming language.
```
Comment (Brenocs86) [inline website_and_docs/content/documentation/webdriver/getting_started/install_selenium_library.pt-br.md:7]: Configurando a biblioteca Selenium para sua linguagem de programção favorita

Comment (Brenocs86) [reply website_and_docs/content/documentation/webdriver/getting_started/install_selenium_library.pt-br.md:7]: translation to portuguese from the "Setting up the Selenium library for your favourite programming language."

**Pull request merged**